### DISCUSSION TITLE
Whats this error execution reverted 2

### DISCUSSION THREAD

**Discussion by EhigieO** [web3j/web3j#1769] [Q&A]
org.web3j.protocol.exceptions.TransactionException: Transaction 0xf727bfa1788388175befe9156e407dd6a5f71dfb9c6a59ec1102fe7dd3d93dfb has failed with status: 0x0. Gas used: 35947. Revert reason: 'execution reverted: 2'.

**Comment by wlswo** [comment]
Did you use token transfer?? 
The specific error cannot be known from the transaction hash value

**Reply by EhigieO** [reply]
I realized I needed to approve access to muy wallet first before the contract can execute. Thanks
